packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -27,6 +28,12 @@ type DropdownProps = {|
      */
     items: Array<DropdownItem>,
 
+    /**
+     * A handler to set the searchText of the parent. When this is provided,
+     * a search text input will be displayed at the top of the dropdown body.
+     */
+    handleSearchTextChanged: ?(e: SyntheticInputEvent<>) => void,
```

Comment:
This should be `onSearchTextChanged` or preferably `onSearchTextChanged?: (searchText: string) => mixed`.  I think we should avoid passing the raw event back.  We're always going to want to extract the text from the input value so DropdownCore may as well do that before passing it back to.

Reply:
This got me thinking about whether we should change the `onChange: (selectedValues: Array<string>) => mixed,` in multi-select.js in the future to be `onSelectionChange` or `onSelectionChanged`. 🤔
No action necessary for this PR.